### PR TITLE
rephrase time expectations

### PR DESCRIPTION
**Pull request by borisonr**
still not perfect, but the hope is that this helps candidates feel more confident in submitting if they're taking a long time to complete the takehome

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We really appreciate you applying to work at Codecademy!
 
 This README.md details the prompt for your assignment.
 
-You can see the competencies we'll be grading it on [here](https://github.com/Codecademy/engineering-competencies/blob/master/interviews/backend-takehome.md). We expect this to take you roughly 2-3 hours. Please let us know if there's anything unclear about it!
+You can see the competencies we'll be grading it on [here](https://github.com/Codecademy/engineering-competencies/blob/master/interviews/backend-takehome.md). On average it takes candidates around 5 hours to complete this challenge, but no worries if it takes you longer. We’ve had successful candidates report taking 12 hours to complete the requirements. Please let us know if there's anything unclear about it!
```
Comment (kipprice) [inline backend/README.md:9]: Thoughts on "successful candidates report taking more than ten hours" instead of calling out 12? I could be sold either way.

**Review by kipprice — APPROVED** [review]
Fine as is, but I left a potential wording tweak.

File: backend/README.md (hunk 1 above)
Comment (borisonr) [reply backend/README.md:9]: that works for me!

Comment (hmafzal) [reply backend/README.md]: If a candidate takes less time, would the new wording make them feel like they did it too quickly? That was one thing I liked about a range. 

Maybe:
We’ve had successful candidates report taking from two to more than ten hours to complete the requirements.

**Review by hmafzal — APPROVED** [review]
Minor wording tweak if we want it

File: backend/README.md (hunk 1 above)
Comment (borisonr) [reply backend/README.md]: i think i'm less concerned about people being too fast... obviously generalizing but i think those folks tend to have less imposter syndrome? (at least speaking from personal experience from when i've finished takehomes quickly), and i worry that even adding 2 hours could stress some folks out?

**Review by JoshuaKGoldberg — APPROVED** [review]

**Review by anaharris — APPROVED** [review]

**Pull request merged**